src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Install the New Relic Extension On Azure Arc
+tags:
+  - Infrastructure
+  - Install NewRelic Extension
+  - NewRelic Extension
+  - Azure Arc Extension
+metaDescription: New Relic's Extension For Azure Arc.
+---
+
+This documentation provides step-by-step instructions on how to install the New Relic Extension on an Azure Arc Connected Machine using the Azure Command-Line Interface (Azure CLI).
+
+## Prerequisites
+
+Before you begin, ensure that you have:
+
+- Azure CLI installed on your local machine.
+- Azure Arc Connected Machine set up and registered in your Azure subscription.
+- The necessary permissions to manage resources in the specified resource group.
```

Comment:
Does this vary by user? Can we give them guidance on those permissions?

Reply:
No, it required same permission which user had while creating the resource.

Reply:
@rahul188 can you be more specific about this? what type of permissions are those?

Reply:
@homelessbirds , it should be part of Azure Subscription Owner or Contributor Role.